easybuild/easyconfigs/h/HDF5/HDF5-1.14.0-GCCcore-12.2.0-serial.eb

```diff
@@ -0,0 +1,26 @@
+name = 'HDF5'
+version = '1.14.0'
+versionsuffix = '-serial'
+
+homepage = 'https://portal.hdfgroup.org/display/support'
+description = """HDF5 is a data model, library, and file format for storing and managing data.
+ It supports an unlimited variety of datatypes, and is designed for flexible
+ and efficient I/O and for high volume and complex data."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
```

Comment:
I don't think it makes sense to drop this all the way to `GCCcore` even for a `serial` version, I think `GCC`/`intel-compilers` would be a better fit

Reply:
... and even mandatory due to non-compatibility of Fortran modules between Intel/GCC?

Reply:
indeed